outbreak_distribution.R

```diff
@@ -0,0 +1,246 @@
+# HPRU script
+
+## Outbreak distribution size
```

Comment:
I do not have further edit suggestion. possibly, could you add a documentation link too?

Reply:
Thanks! I'll add the links in a bit once I've finished reviewing each demo.